daffodil-tdml/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -89,44 +81,54 @@ import org.apache.daffodil.processors.charset.BitsCharsetNonByteSize
 import org.apache.daffodil.processors.charset.BitsCharsetNonByteSizeEncoder
 import org.apache.daffodil.io.FormatInfo
 import org.apache.daffodil.io.InputSourceDataInputStream
-import java.nio.charset.{ Charset => JavaCharset }
+import java.nio.charset.{Charset => JavaCharset}
 import java.io.OutputStream
 
+import org.apache.daffodil.tdml.processor.{
+  TDMLParseResult,
+  DaffodilTDMLDFDLProcessor,
+  TDMLDFDLProcessor,
+  TDMLResult,
+  TDMLUnparseResult,
+  TDML,
+  TDMLDFDLProcessorFactory
+}
+
 /**
- * Parses and runs tests expressed in IBM's contributed tdml "Test Data Markup Language"
- */
+  * Parses and runs tests expressed in IBM's contributed tdml "Test Data Markup Language"
+  */
```

Comment:
Eclipse is failing badly for me since the upgrades to scala 2.12. Eclipse oxygen with the scala plugin is quite unstable when working with Daffodil.

I've had to switch to IntelliJ IDEA, which is more stable but has its own funny stuff.  There's also an entirely new set of key bindings to learn, and I mistype lots of them resulting in strange actions that don't seem like they did anything but may have. 

So I will try to revert things like this that are just whitespace massaging, and to find and disable the IDEA features causing this to match prior style.

In general, our code-style things like use of spaces vs. tabs, are all getting clobbered these days. Lots of XML files are now full of tabs. I don't think this is my fault, as I do have IDEA setup for spaces, not tabs.

Reply:
I have a new eclipse environment, that is working better now.  Somehow loading the whole Scala IDE, which is eclipse with the scala components bundled into it, works much better than downloading Eclipse oxygen, and then applying the scala IDE to it. MUCH better.